src/components/navigation/nav-tab-buttons.tsx

```diff
@@ -58,7 +58,7 @@ export class NavTabButtons extends BaseComponent<IProps, IState> {
       ui.setActiveNavTab(tab);
       logEvent();
     }
-    ui.toggleNavTabContent(true);
+    // ui.toggleNavTabContent(true);
```

Comment:
Is this commented code needed?

Reply:
It turns out the entire component is no longer needed so I removed it.